spark-commons/src/main/scala/za/co/absa/spark/commons/implicits/DataFrameImplicits.scala

```diff
@@ -37,9 +37,8 @@ object DataFrameImplicits {
       dfData
     }
 
-    //TODO  Fix ScalaDoc cross-module links #48 - Dataset.show()
     /**
-     * Get the string representation of the data in the format as Dataset.show() displays them
+     * Get the string representation of the data in the format as [[org.apache.spark.sql.Dataset#show():Unit* spark.Dataset.show()]] displays them
```

Comment:
This row do not navigate to defined method. Only to class.

Reply:
No idea how to improve on it. 🤷‍♂️ 
Once released, I want to see, if the link works in code navigation. 🤞